src/app.js

```diff
@@ -6,8 +6,223 @@ import _ from 'underscore';
 import './css/foundation.css';
 import './css/style.css';
 
-console.log('it loaded!');
+import Trip from './app/models/trip';
+import TripList from './app/collections/trip_list';
 
-$(document).ready( () => {
-  $('main').html('<h1>Hello World!</h1>');
+const tripList = new TripList();
+tripList.fetch();
+
+let tripTemplate;
+const renderTrips = function renderTrips(tripList) {
+  $('.reservation-form').hide();
+  $('#trip-info').hide();
+  $('#trip-list').empty();
+  $('#filter-form').show();
+
+  tripList.forEach((trip) => {
+    $('#trip-list').append(tripTemplate(trip.attributes));
+  });
+  $('#trips-table').show();
+};
+
+let singleTripTemplate;
+const renderOneTrip = function renderOneTrip(id) {
+  $('#trip-info').empty();
+  $('#trip-info').show();
+  $('#new-trip-form').hide();
+  $('.reservation-form').show();
+
+  let trip;
+  trip = new Trip({id: id});
+  trip.fetch().done(() => {
+    $('#trip-info').append(singleTripTemplate(trip.attributes));
+  });
+};
+
+const tripFields = ['name', 'cost', 'weeks', 'continent', 'about', 'category'];
+
+const events = {
+  addTrip(event) {
+    event.preventDefault();
+    const tripData = {};
+    tripFields.forEach( (field) => {
```

Comment:
This whole getting the trip fields could use being put into a separate function.